inspirehep/dojson/experiments/fields/bd1xx.py

```diff
@@ -36,11 +36,26 @@
 @utils.for_each_value
 def experiment_names(self, key, value):
     """Experiment names."""
-    if value.get('u'):
+
+    name = value.get('u')
```

Comment:
Take a look at the newly added `force_single_element` helper: https://github.com/inspirehep/inspire-next/blob/1f1774933e711982f884ccd7a45cb01daeccb851/inspirehep/dojson/utils/__init__.py#L99-L104, and see if you can benefit from it.